openmm-beta/bld.bat

```diff
@@ -0,0 +1,31 @@
+mkdir build
+cd build
+
+cmake.exe .. -G "NMake Makefiles JOM" ^
+    -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%" ^
+    -DCMAKE_PREFIX_PATH="%LIBRARY_PREFIX%" ^
+    -DCMAKE_BUILD_TYPE=Release ^
+    -DBUILD_TESTING=OFF ^
+    -DCUDA_TOOLKIT_ROOT_DIR="C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v%CUDA_VER%" ^
+    -DOPENCL_INCLUDE_DIR="C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v%CUDA_VER%/include" ^
+    -DOPENCL_LIBRARY="C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v%CUDA_VER%/lib/x64/OpenCL.lib" ^
```

Comment:
Do we also need to set `FFTW_INCLUDES` and `FFTW_LIBRARY`?  See

https://github.com/openmm/openmm/blob/cc3c4b54305fd53b535ddf26df15d9e1bb903e47/devtools/packaging/scripts/windows/build.bat#L6-L8

Reply:
Since it's provided as part of the `%PREFIX%` environment (conda-forge's fftw), I was assuming that CMake will find it automatically if it's not complaining. Would the compilation still proceed without fftw?

Reply:
Yes, but the OpenMMPME plugin wouldn't get built.  Hence my suggestion to add a check for that plugin.

Reply:
It's [being found in the patching step after the build](https://dev.azure.com/OmniaMD/conda-dev-recipes/_build/results?buildId=339&view=logs&j=4bc7ef66-8a59-503f-fa7a-b9533d6853d5&t=87af48f0-e125-5674-9c51-188fbf8597de&l=12042), so it is getting built. I'll add the flags anyway to be consistent with `build.sh`.